### DISCUSSION TITLE
When will the next version be made GA?

### DISCUSSION THREAD

**Discussion by tonyvperez** [py-pdf/pypdf#2703] [Q&A] [answered]
There is a compatibility issue with Acrobat that is addressed with this fix:  https://github.com/py-pdf/pypdf/pull/2614

When will it be made available?

**Comment by stefan6419846** [comment]
There are no explicit plans for releases. In the best case, we would merge a fix for #2697 beforehand and schedule a release for this weekend.

**Comment by pubpub-zz** [comment, marked as answer]
You can also do an installation from a pre-release version as described inhere:
https://pypdf.readthedocs.io/en/stable/dev/testing.html#evaluate-a-pr-in-progress-version
